package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "fvtt-module-vueport",
+    "type": "module",
+    "version": "1.2.0",
```

Comment:
We should keep this in sync with the version in `module.json`, which is still 1.1.

I suggest we should only increment the version numbers in the final commit for a release (which would get tagged as well so the manifest and download links are valid).